hm-basic-webapp/templates/service.yaml

```diff
@@ -3,6 +3,7 @@ kind: Service
 metadata:
   name: {{ include "hm-basic-webapp.prefix" . }}
   labels: {{ toYaml .Values.serviceLabels | nindent 8 }}
+  annotations: {{ toYaml .Values.serviceAnnotations | nindent 8 }}
```

Comment:
I think they should be 4 not 8. @slashmili WDYT?

```
# Source: sample-app/charts/hm-basic-webapp/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: develop-sample-app
  labels:
        foo: bar
        service-discovery-name: sample-app
  annotations:
        foo: bar
        some_key: some_value
spec:
  ports:
    - name: http
      port: 80
      targetPort: 9292
  selector:
    app: develop-sample-app
  type: ClusterIP
```

Reply:
yup, the rest of the yaml has 2 space tabs. so 4 is correct. I think I copied label line from somewhere else.

Reply:
✨ After the change from 4 to 8:

```
# Source: sample-app/charts/hm-basic-webapp/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: develop-sample-app
  labels: 
    foo: bar
    service-discovery-name: sample-app
  annotations: 
    foo: bar
    some_key: some_value
spec:
  ports:
    - name: http
      port: 80
      targetPort: 9292
  selector:
    app: develop-sample-app
  type: ClusterIP
  ```